homeassistant/config.py

```diff
@@ -463,22 +466,15 @@ def async_check_ha_config_file(hass):
 
     This method is a coroutine.
     """
-    import homeassistant.components.persistent_notification as pn
-
     proc = yield from asyncio.create_subprocess_exec(
-        sys.argv[0],
-        '--script',
-        'check_config',
-        stdout=asyncio.subprocess.PIPE)
+        sys.executable, '-m', 'homeassistant', '--script',
```

Comment:
`sys.executable -m homeassistant` is not necessarily running the same executable.

It will run what was installed with `pip` even if another (cloned from git) version is running.